Description: add encoding directive
 otherwise we get a POD ERROR:
       Around line 29:
           Non-ASCII character seen before =encoding in 'Hellesoy'. Assuming UTF-8
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-06-07

--- a/lib/Test/BDD/Cucumber/I18N/Data.pm
+++ b/lib/Test/BDD/Cucumber/I18N/Data.pm
@@ -5,6 +5,7 @@
 
 use JSON::MaybeXS qw/decode_json/;
 
+=encoding UTF-8
 
 =head1 NAME
 
